pkg/bundle/bundle.go

```diff
@@ -244,3 +244,14 @@ func (b *bundle) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result,
 
 	return ctrl.Result{}, b.client.Status().Update(ctx, &bundle)
 }
+
+func (b *bundle) bundleTargetNamespaceSelector(bundleObj *trustapi.Bundle) (labels.Selector, error) {
+	namespaceSelector := labels.Everything()
```

Comment:
Nit: Is this variable needed? Why not just inline it?

Reply:
I updated the code, also made some more changes. PTAL